NuGet.Config

```diff
@@ -19,6 +19,7 @@
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
     <add key="MyGetApplicationInsights" value="https://www.myget.org/F/applicationinsights" />
+    <add key="dotnet" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" protocolVersion="3" />
```

Comment:
I am getting the following error

`Unable to resolve 'System.Diagnostics.DiagnosticSource (>= 4.4.0)' for '.NETStandard,Version=v1.3'`

DiagnosticSource ver. 4.4 is not on official NuGet yet. I believe this is related to the new correlation/automatic dependency tracking and brought onto us by the new Microsoft.ApplicationInsights.dll